packages/patternfly-4/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/EmptyState/empty-state.css';
+
+export enum EmptyStateVariant {
+  large = 'large',
+  small = 'small',
+  full = 'full'
+}
+
+export interface EmptyStateProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the EmptyState */
+  className?: string;
+  /** Content rendered inside the EmptyState */
+  children: React.ReactNode;
+  /** Modifies EmptyState max-width */
+  variant?: EmptyStateVariant;
+}
+
+const maxWidthModifiers = {
+  large: styles.modifiers.lg,
+  small: styles.modifiers.sm,
+  full: null
+}
+
+export const EmptyState: React.FunctionComponent<EmptyStateProps> = ({
+  children,
+  className = '',
+  variant = EmptyStateVariant.large,
+  ...props
+}) => {
+  const maxWidthModifier = variant && maxWidthModifiers[variant];
+  return (
+    <div className={css(styles.emptyState, maxWidthModifier && maxWidthModifier, className)} {...props}>
```

Comment:
I know you didn't write this logic, but please change `maxWidthModifier && maxWidthModifier` to just `maxWidthModifier`. Anything `&&` with itself is itself.